### ISSUE TITLE
Not re-genarate the bundle tar.gzip file when retyr a PP

### ISSUE THREAD

**Issue by freddyucv** [dotCMS/core#20878]
**Describe the bug**
After the PP performance chance in release-21.05, when try to retry a PP bundle the tar.gzip file is re-generate again.

**Expected behavior**
The tar.gzip file should not be re-generate when a PP bundle is retrying

**Comment by victoralfaro-dotcms** [comment]
First attemtp to PP:

```
17:10:58.910  WARN  pool.HikariPool - jdbc/dotCMSPool - Thread starvation or clock leap detected (housekeeper delta=1m27s148ms).
17:10:58.910  WARN  pool.HikariPool - HikariPool-1 - Thread starvation or clock leap detected (housekeeper delta=1m8s84ms).
17:10:58.910  WARN  pool.HikariPool - HikariPool-2 - Thread starvation or clock leap detected (housekeeper delta=1m8s89ms).
17:11:05.647  INFO  publishing.PublisherAPIImpl - Writing bundle.xml file
17:11:20.552  INFO  publishing.PublisherAPIImpl - Start of Bundler: DependencyBundler
17:11:21.397  INFO  caffine.CaffineCache - ***	 Building Cache : pushedassetscache, size:5000,Concurrency:32
17:11:24.048  INFO  dependencies.ConcurrentDependencyProcessor - DependencyProcessor Finished
17:11:26.421  INFO  publishing.PublisherAPIImpl - End of Bundler: DependencyBundler
17:11:28.246  INFO  publishing.PublisherAPIImpl - Start of Bundler: HostBundler
17:11:28.897  INFO  util.PushPublishLogger - class com.dotcms.enterprise.publishing.remote.bundler.HostBundler : Status Update: Bundling.
17:11:28.969  INFO  util.PushPublishLogger - class com.dotcms.enterprise.publishing.remote.bundler.HostBundler : Status Update: Bundling.
17:11:31.085  INFO  publishing.PublisherAPIImpl - End of Bundler: HostBundler
17:11:31.086  INFO  publishing.PublisherAPIImpl - Start of Bundler: ContentBundler
17:11:31.119  INFO  util.PushPublishLogger - class com.dotcms.enterprise.publishing.remote.bundler.ContentBundler : Status Update: Bundling.
17:11:31.246  INFO  util.PushPublishLogger - class com.dotcms.enterprise.publishing.remote.bundler.ContentBundler : Status Update: Bundling.
17:11:31.296  INFO  publishing.PublisherAPIImpl - End of Bundler: ContentBundler
17:11:31.296  INFO  publishing.PublisherAPIImpl - Start of Bundler: FolderBundler
17:11:31.297  INFO  publishing.PublisherAPIImpl - End of Bundler: FolderBundler
17:11:31.298  INFO  publishing.PublisherAPIImpl - Start of Bundler: TemplateBundler
17:11:31.298  INFO  publishing.PublisherAPIImpl - End of Bundler: TemplateBundler
17:11:31.299  INFO  publishing.PublisherAPIImpl - Start of Bundler: ContainerBundler
17:11:31.299  INFO  publishing.PublisherAPIImpl - End of Bundler: ContainerBundler
17:11:31.299  INFO  publishing.PublisherAPIImpl - Start of Bundler: RuleBundler
17:11:31.300  INFO  publishing.PublisherAPIImpl - End of Bundler: RuleBundler
17:11:31.300  INFO  publishing.PublisherAPIImpl - Start of Bundler: LinkBundler
17:11:31.301  INFO  publishing.PublisherAPIImpl - End of Bundler: LinkBundler
17:11:31.301  INFO  publishing.PublisherAPIImpl - Start of Bundler: ContentTypeBundler
17:11:31.302  INFO  publishing.PublisherAPIImpl - End of Bundler: ContentTypeBundler
17:11:31.302  INFO  publishing.PublisherAPIImpl - Start of Bundler: RelationshipBundler
17:11:31.303  INFO  publishing.PublisherAPIImpl - End of Bundler: RelationshipBundler
17:11:31.303  INFO  publishing.PublisherAPIImpl - Start of Bundler: LanguageVariablesBundler
17:11:31.324  INFO  publishing.PublisherAPIImpl - End of Bundler: LanguageVariablesBundler
17:11:31.325  INFO  publishing.PublisherAPIImpl - Start of Bundler: WorkflowBundler
17:11:31.325  INFO  publishing.PublisherAPIImpl - End of Bundler: WorkflowBundler
17:11:31.325  INFO  publishing.PublisherAPIImpl - Start of Bundler: LanguageBundler
17:11:31.373  INFO  util.PushPublishLogger - class com.dotcms.enterprise.publishing.remote.bundler.LanguageBundler : Status Update: Bundling.
17:11:31.420  INFO  publishing.PublisherAPIImpl - End of Bundler: LanguageBundler
17:11:31.421  INFO  publishing.PublisherAPIImpl - Start of Bundler: BundleXMLAsc
17:11:31.421  INFO  publishing.PublisherAPIImpl - End of Bundler: BundleXMLAsc
17:11:31.422  INFO  publishing.PublisherAPIImpl - Start of Bundler: CategoryBundler
17:11:31.422  INFO  publishing.PublisherAPIImpl - End of Bundler: CategoryBundler
```
Next attempt to PP

```
17:11:42.925  INFO  util.PushPublishLogger - class com.dotcms.publisher.business.PublisherQueueJob : Status update: Failed to update bundle audit status.
17:11:42.968  INFO  business.PublisherQueueJob - ===========================================================
17:11:42.968  INFO  business.PublisherQueueJob - For bundle '01FEPW7YNR4CE69YW21RM7DS50':
17:11:42.968  INFO  business.PublisherQueueJob - -> Status             : FAILED_TO_SEND_TO_ALL_GROUPS [4]
17:11:42.968  INFO  business.PublisherQueueJob - -> Re-publish attempts: 1 out of 3
17:11:42.968  INFO  business.PublisherQueueJob - ===========================================================
17:11:42.968  INFO  business.PublisherQueueJob -
17:11:42.968  INFO  business.PublisherQueueJob - Found 1 bundle(s) to process.
17:11:42.968  INFO  business.PublisherQueueJob -
17:11:42.968  INFO  business.PublisherQueueJob - Processing bundle: ID: 01FEPW7YNR4CE69YW21RM7DS50. Status: 4. Publish Date: 2021-09-03 17:08:00.0
17:11:42.968  INFO  util.PushPublishLogger - class com.dotcms.publisher.business.PublishAuditStatus : Status Update: Pending.
17:11:42.968  INFO  util.PushPublishLogger - class com.dotcms.publisher.business.PublisherQueueJob : Pre-publish work started.
[class com.dotcms.publisher.pusher.PushPublisher]
17:11:42.970  INFO  util.PushPublishLogger - class com.dotcms.publisher.business.PublisherQueueJob : Pre-publish work complete.
17:11:42.970  INFO  util.PushPublishLogger - class com.dotcms.publishing.PublisherAPIImpl : [BundleID: 01FEPW7YNR4CE69YW21RM7DS50] Started Publishing Task
17:11:56.295  INFO  publishing.PublisherAPIImpl - Retrying bundle: 01FEPW7YNR4CE69YW21RM7DS50, we don't need to run bundlers again
```
See last line ☝🏼

**Comment by bryanboza** [comment]
Fixed, tested on release-21.10. // Docker // FF

**Comment by jcastro-dotcms** [comment]
Excluding fix from dotCMS 5.3.8.7 LTS release as most of the changes relay on code added via ticket #20010 : `Add Manifest to Bundle`, which was not back-ported either due to extensive code changes.
